### PR TITLE
chore: update js snippet in the sample app

### PR DESCRIPTION
**Pull request by 1abhishekpandey**
[Linear ticket](https://linear.app/rudderstack/issue/SDK-2142/update-flutter-web-snippet)

## Description of the change

> Updated the code snippet for the Flutter Web. Now it points to the JS SDK v3 initialisation snippet.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: packages/example/web/index.html [hunk 1]
```diff
@@ -31,19 +31,31 @@
...
+      }}(n)}try{new Function('return import("")'),window.rudderAnalyticsBuildType="modern"}catch(a){}
+      var i=document.head||document.getElementsByTagName("head")[0]
+      ;var d=document.body||document.getElementsByTagName("body")[0];window.rudderAnalyticsAddScript=function(e,t,r){
+      var n=document.createElement("script");n.src=e,n.setAttribute("data-loader","RS_JS_SDK"),t&&r&&n.setAttribute(t,r),
+      "async"===scriptLoadingMode?n.async=true:"defer"===scriptLoadingMode&&(n.defer=true),
+      i?i.insertBefore(n,i.firstChild):d.insertBefore(n,d.firstChild)},window.rudderAnalyticsMount=function(){
+      "undefined"==typeof globalThis&&(Object.defineProperty(Object.prototype,"__globalThis_magic__",{get:function get(){
+      return this},configurable:true}),__globalThis_magic__.globalThis=__globalThis_magic__,
+      delete Object.prototype.__globalThis_magic__),
+      window.rudderAnalyticsAddScript("".concat(sdkBaseUrl,"/").concat(window.rudderAnalyticsBuildType,"/").concat(sdkName))
```
Comment (saikumarrs) [inline packages/example/web/index.html:53]: It is better to have this also.
Can you update the README file to replace this value with the actual write key?

```suggestion
      window.rudderAnalyticsAddScript("".concat(sdkBaseUrl,"/").concat(window.rudderAnalyticsBuildType,"/").concat(sdkName), "data-rsa-write-key", "<WRITE_KEY>")
```

Comment (1abhishekpandey) [reply packages/example/web/index.html]: I've added the proposed changes in the `index.html` file. But what exactly do I need to update in the readme?

Comment (saikumarrs) [reply packages/example/web/index.html]: Let's add a step to replace this placeholder write key with the actual value.

Comment (1abhishekpandey) [reply packages/example/web/index.html]: Are you suggesting replacing the placeholder writeKey in the `index.html` itself, right? If that's the case then I need to research it.

Comment (saikumarrs) [reply packages/example/web/index.html]: https://github.com/rudderlabs/rudder-sdk-flutter/blob/develop/packages/example/README.md

```
2. To deploy the "web" package, ensure to replace the `<WRITE_KEY>` in `/web/index.html` with the actual write key value.
```

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**